### PR TITLE
add rather hacky border to avatars

### PR DESCRIPTION
**Pull request by lily-pytel**
~This is real hacky... but it's a thing. The alternative of building hexagons in photoshop to match the avatars might be better~

Never mind that hacky stuff

### CODE REVIEW

File: next/components/user-avatar.js [hunk 1]
```diff
@@ -1,31 +1,115 @@
 import PropTypes from 'prop-types';
+import { RARITY_COLORS } from '../constants/rarities';
```
[the remark this answers is not in the document]
Comment (lily-pytel) [reply next/components/user-avatar.js:2]: We do use this thing for the deck builder as well, to show the rarities of cards so I'll leave it for now

File: next/components/user-avatar.js [hunk 2]
```diff
@@ -1,31 +1,115 @@
...
+  profileIconId,
+  accountType = 'BASIC',
+  small = false
+}) {
   const avatarId = profileIconId || 1;
   const safeAvatarId = Math.ceil(Math.floor(avatarId, 62));
-  const className = `profile-image ${small && 'small'}`;
+  const borderColor =
+    accountType && accountType !== 'BASIC'
+      ? RARITY_COLORS[accountType.toLowerCase()]
```
[the remark this answers is not in the document]
Comment (lily-pytel) [reply next/components/user-avatar.js:15]: Maybe I'll just combine these two comments and make a ACCOUNT_TYPES_COLORS that takes uppercase lol

File: next/components/user-avatar.js [hunk 3]
```diff
@@ -1,31 +1,115 @@
...
+      : '';
+  const backgroundClassName = `${borderColor ? 'hexagon' : 'n'} ${borderColor &&
```
[the remark this answers is not in the document]
Comment (lily-pytel) [reply next/components/user-avatar.js:17]: Ooh yeah, that is much nicer

File: next/components/user-avatar.js [hunk 4]
```diff
@@ -1,31 +1,115 @@
...
 
   return (
-    <>
+    <div className="container">
       <style jsx>{`
+        .container {
+          position: relative;
+          width: ${width};
+          height: ${height};
+          margin-top: ${small ? '0' : '30px'};
```
[the remark this answers is not in the document]
Comment (lily-pytel) [reply next/components/user-avatar.js:30]: To me it's a bit clearer this way but will change

**Comment by alexmiddeleer** [comment]
I mean, it might be better. I'm not used jsx and don't know the best
practices.

**Comment by lily-pytel** [comment]
Sorry for all the specific css... I think it's inevitable

File: next/components/user-avatar.js [hunk 5]
```diff
@@ -1,35 +1,81 @@
...
+  profileIconId,
+  accountType = 'BASIC',
+  small = false
+}) {
   const avatarId = profileIconId || 1;
   const safeAvatarId = Math.ceil(Math.floor(avatarId, 62));
-  const className = `profile-image ${small && 'small'}`;
+  const containerClassNames = `container ${small ? 'small' : 'large'}`;
+  const borderImage =
+    (accountType && ACCOUNT_BORDERS[accountType]) || ACCOUNT_BORDERS['BASIC'];
```
Comment (alexmiddeleer) [inline next/components/user-avatar.js:15]: technically I guess you could do ACCOUNT_BORDERS.BASIC but prettier might flip a desk.

**Pull request merged**